# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: toolkit/components/antitracking/ContentBlockingNotifier.h
# Commit: 75beefd331e4
# Full Hash: 75beefd331e41a202e652d4427a58a3e3d207762
# Author: Dimi Lee <dlee@mozilla.com>
# Date: 2020-04-08 21:42:38
# Regressor Bug: 1624552
# File Overlap Count: 1
# Description:
#   Bug 1624552 - P2. Merge reporting channel and tracking channel for ContentBlockingNotifier::OnEvent r=timhuang,baku
#   
#   The only place is using different the reporting channel and the tracking channel
#   is ContentBlocking::AllowAccessFor, which use the top-window of the
#   parent window (reporting) and the parent window (tracking).
# ==============================================================================

diff -r 277c9d5c6e33 -r 75beefd331e4 toolkit/components/antitracking/ContentBlockingNotifier.h
--- a/toolkit/components/antitracking/ContentBlockingNotifier.h	Wed Apr 08 10:04:10 2020 +0000
+++ b/toolkit/components/antitracking/ContentBlockingNotifier.h	Wed Apr 08 10:04:23 2020 +0000
@@ -52,8 +52,7 @@
   static void OnEvent(nsIChannel* aChannel, uint32_t aRejectedReason);
 
   static void OnEvent(
-      nsIChannel* aReportingChannel, nsIChannel* aTrackingChannel,
-      bool aBlocked, uint32_t aRejectedReason,
+      nsIChannel* aChannel, bool aBlocked, uint32_t aRejectedReason,
       const nsACString& aTrackingOrigin,
       const Maybe<StorageAccessGrantedReason>& aReason = Nothing());
 
